hypervisor/dm/vpci/vpci.c

```diff
@@ -617,7 +617,7 @@ struct pci_vdev *vpci_init_vdev(struct acrn_vpci *vpci, struct acrn_vm_pci_dev_c
 	if (dev_config->vdev_ops != NULL) {
 		vdev->vdev_ops = dev_config->vdev_ops;
 	} else {
-		if (get_highest_severity_vm(false) == vpci2vm(vpci)) {
+		if ((PRE_VM_NUM == 0U) && get_highest_severity_vm(false) == vpci2vm(vpci)) {
```

Comment:
If we have pre-launched VM, the severity of the VM is always higher than SOS VM, right?

Reply:
Check base on severity is better than this

Reply:
For now, yes, it's true. But do we need to support pre-launched VM which severity would lower than the SOS ?

Reply:
at moment, pre-launched vm is for rtvm or safety vm.